modin/core/storage_formats/base/query_compiler.py

```diff
@@ -1768,24 +1768,33 @@ def to_timedelta(self, unit="ns", errors="raise"):  # noqa: PR02
             self, unit=unit, errors=errors
         )
 
-    # FIXME: get rid of `**kwargs` parameter (Modin issue #3108).
-    @doc_utils.add_one_column_warning
-    @doc_utils.add_refer_to("Series.unique")
-    def unique(self, **kwargs):
+    @doc_utils.add_refer_to("Series.drop_duplicates")
```

Comment:
```suggestion
    @doc_utils.add_refer_to("Series.unique")
```

Reply:
this was done on purpose, the `qc.unique()` method now uses most of the arguments from `Series.drop_duplicates()` so it's makes sense to refer to this method instead

Reply:
Since we want to have a stable QC API, it seems that having this kind of mismatch is not a good idea. I would also like to have different QCs. One is responsible for d2p, second is as a base class for descendants. BaseQC does both the things now, which is not pretty good I think. What do you think?

Reply:
> Since we want to have a stable QC API, it seems that having this kind of mismatch is not a good idea.

So I can rename `qc.unique -> qc.drop_duplicates`, would that be better?

> I would also like to have different QCs. One is responsible for d2p, second is as a base class for descendants.

Looks like unnecessary overcomplication to me. I don't see a problem in making the base class to implement its methods as d2p. I believe all the backends would inherit from d2p qc and ignore the abstract one, what's the point then in having and supporting both of them?

Reply:
> So I can rename qc.unique -> qc.drop_duplicates, would that be better?

It seems we already have drop_duplicates in the base qc. Maybe just put a comment for now on why we refer to drop_duplicates docs?

> Looks like unnecessary overcomplication to me. I don't see a problem in making the base class to implement its methods as d2p. I believe all the backends would inherit from d2p qc and ignore the abstract one, what's the point then in having and supporting both of them?

I am talking about d2p and base qc, which is not an abstract but rather common functionality for all storage formats. Would it make sense? It looks like the base qc does double work at the moment.

Reply:
> It seems we already have drop_duplicates in the base qc. Maybe just put a comment for now on why we refer to drop_duplicates docs?

No, there's no `drop_duplicates()` method on the qc level, only `.duplicated()`. Will put a comment though on why we're referring to drop duplicates

Reply:
> I am talking about d2p and base qc, which is not an abstract but rather common functionality for all storage formats. Would it make sense? It looks like the base qc does double work at the moment.

I don't know, I don't see why that would be a problem and why we should support two QCs for that matter